arm9/source/utils/scripting.c

```diff
@@ -259,86 +265,125 @@ static inline u32 hexntostr(const u8* hex, char* str, u32 len) {
     return len;
 }
 
-static inline u32 line_len(const char* text, u32 len, u32 ww, const char* line, char** eol) {
-    u32 last = ((text + len) - line);
+static inline bool is_crlf(const char* str) {
```

Comment:
Majority rule seems to make the most sense here... We _could_ do a hotkey to switch newline types on the keyboard, and display the current newline type status on the keyboard screen too, let me know if you think that's a good idea.